### PR TITLE
Completed css-fonts.

### PR DESCRIPTION
**Pull request by KrisMackenroth**
Screenshot (237)
Screenshot (236)
Screenshot (235)

### CODE REVIEW

File: css-fonts/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Macondofamily=Roboto:wght@100display=swap" rel="stylesheet">
```
Comment (TimelikeClosure) [inline css-fonts/index.html:7]: Your Google font URL is mangled and isn't loading the fonts correctly:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: css-fonts/index.html (hunk 1 above)
Comment (KrisMackenroth) [reply css-fonts/index.html]: Not too sure how to fix it. I broke it into two different links to try and not mangle it.

File: css-fonts/styles.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    sans-serif;
+}
+
+h2 {
+    font-family: 'Macondo',
+    cursive;
+    font-family: 'Roboto',
+    sans-serif;
+
+}
```
Comment (TimelikeClosure) [inline css-fonts/styles.css:21]: Make sure both Google Fonts are displayed somewhere on the page at the same time. Right now, `Macondo` is being overwritten by `Roboto` everywhere.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: css-fonts/index.html (hunk 1 above)
Comment (TimelikeClosure) [reply css-fonts/index.html]: @KrisMackenroth go back and grab the URL from the Google Fonts site again.

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**